### PR TITLE
new icon: jiraalign (original,plain,original-wordmark,plain-wordmark)

### LINKED ISSUE #789: Icon request: JiraAlign
### About the icon
JiraAlign is software development SDLC tool that closely integrates with Jira.

### Links
https://wac-cdn.atlassian.com/dam/jcr:b821902e-5ee6-432f-a7e2-9f18aaa2641b/Jira%20Align-blue-rgb.svg?cdnVersion=1747
https://www.atlassian.com/software/jira/align

### PR DESCRIPTION
**Pull request by yeyeto2788**
## Double check these details before you open a PR

- [X] PR does not match another non-stale PR currently opened
- [X] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [X] PR's base is the `develop` branch.
- [X] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [X] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [X] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #789 **

### CODE REVIEW

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Those don't look like the official logos 🙊 
Please change to the official ones (I linked to them in issue #789)

**Comment by yeyeto2788** [comment]
That is really weird cuz I do see it correctly in Inkscape
image

Any suggestions on how to fix it? Maybe exporting it differently?

**Comment by yeyeto2788** [comment]
I'll grouping them to see if that fixes the issue.

**Comment by yeyeto2788** [comment]
I had to use the ones you provided on the issue itself as the grouping didn't work. Thanks for that!

**Comment by lunatic-fox** [comment]
Hello there!
@yeyeto2788 you're doing great!
I just have some recommendations to the icons.
First, I think you'll need to change the icons name and add two more icons:
* `original` to `original-wordmark`
* `plain` to `plain-wordmark`

And add the `original` and `plain` icons based on the square-like icon recommended by @Snailedlt on #789 issue.

To `original-wordmark` icon you can ungroup all and unify just "Jira Align" characters in one path.
To `plain-wordmark` icon, you can ungroup all and unify all in just one path.
All icons need to be an 128×128px square.

Wish for the best!

**Comment by yeyeto2788** [comment]
Hey @lunatic-fox!

I think I have covered everything you asked me for but if you see there is something missing just let me know.

Best.

**Comment by lunatic-fox** [comment]
The icons look great now @yeyeto2788! 🤩

However there are some issues yet:
* `plain` and `plain-wordmark` paths need to be united in just one path, to keep them as simple as possible. 
**Source doc:** https://github.com/devicons/devicon/wiki/SVG-Standards

* In `devicon.json` the icon color is `#F04E98`, I suggest `#2684FF` since is the main color of the `original` version.

* You can add `plain-wordmark` to `font` array of `versions` object of `devicon.json`.

* A recommendation is to ungroup all since you already fixed the invisible paths issue in your last icons.

Keep rocking! 🚀

**Comment by yeyeto2788** [comment]
Hey @lunatic-fox! 

I hope this time is correct. So for uniting the path I followed these steps:

1. Ungroup the objects `Ctrl Shift G`
2. Select all `Ctrl A`
3. Path > Union `Ctrl + +` 

I guess that could be added into the documentation. I really helped me a lot the example with the box creation and this might as well help others.

**Comment by lunatic-fox** [comment]
Hello @yeyeto2788!
In short:
* `original` ✔
* `original-wordmark` ✔
* `plain` ✔

But `plain-wordmark` is grouped. The key command to ungroup is `Shift+Ctrl+G`, otherwise it'll probably group instead. However, you can also follow that way bellow.

<details>
<summary>Checking layers</summary>

* Go to `Layers and Objects`. If it's not opened you can open it by `Object > Layers and Objects...` or `Shift+Ctrl+L`
* If you see some layer with folder-icon, that means this layer is a group of layers.
* You can ungroup this layer by right clicking on the layer folder-icon and selecting `Ungroup`.
* To be sure that everything is ungrouped, all layers should be with this icon path-icon
* Once all layers are paths path-icon, them select all and `Unify` by `Path > Union` or `Ctrl++`
* Lastly you should see just one path-icon path layer.
</details>

Hope everything works fine! 👍🏼

**Comment by yeyeto2788** [comment]
Hopefully this one is the last one. Once again sorry for the inconveniences.

Best.

**Comment by lunatic-fox** [comment]
No worries! There is no inconvenience at all. 🙂
Like the other PR, I haven't seen before just two things: the `altnames` and `tags` properties are missing after `name` property.

*Suggestions of tags*
```json
"tags": [
    "development",
    "integration",
    "software"
]
```

The icons are great now! 🎉

Like I said before you can also contact us on [**Discord server**](https://discord.gg/hScy8KWACQ).

File: devicon.json [hunk 1]
```diff
@@ -8945,5 +8945,28 @@
         },
         "color": "#F04E98",
         "aliases": []
+    },
+    {
+        "name": "jiraalign",
+        "altnames": [],
```
Comment (Snailedlt) [inline devicon.json:8951]:
```suggestion
        "altnames": [
            "Jira Align"
        ],
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Almost there! 🚀 

Just need to add the name with whitespaces as an altname and minimize all of the icons. You can minimize the icons like I explained here: https://github.com/devicons/devicon/pull/1465/files#r1008766362

**Comment by Snailedlt** [comment]
@yeyeto2788 
> Hey @lunatic-fox!
> 
> I hope this time is correct. So for uniting the path I followed these steps:
> 
> 1. Ungroup the objects `Ctrl Shift G`
> 2. Select all `Ctrl A`
> 3. Path > Union `Ctrl + +`
> 
> I guess that could be added into the documentation. I really helped me a lot the example with the box creation and this might as well help others.

Yes, this could be added to the wiki as a part of this issue: #1306

**Comment by lunatic-fox** [comment]
> @yeyeto2788
> 
> > Hey @lunatic-fox!
> > I hope this time is correct. So for uniting the path I followed these steps:
> > 
> > 1. Ungroup the objects `Ctrl Shift G`
> > 2. Select all `Ctrl A`
> > 3. Path > Union `Ctrl + +`
> > 
> > I guess that could be added into the documentation. I really helped me a lot the example with the box creation and this might as well help others.
> 
> Yes, this could be added to the wiki as a part of this issue: #1306

I'll be adding this later [Instructions to resize to square](https://github.com/devicons/devicon/pull/1465#issuecomment-1287950577) and [Checking layers](https://github.com/devicons/devicon/pull/1468#issuecomment-1290702218).

**Comment by Snailedlt** [comment]
@lunatic-fox Nice! Please add links to those as comments with the same format I used in the issue I created: #1306

**Comment by Snailedlt** [comment]
> Hi!
> 
> I'm the `check-bot` and we have some issues with your PR:
> 
> ```
> SVG Error in 'jiraalign-plain.svg':
> - 'viewBox' is not '0 0 128 128' -> Set it or scale the file using https://www.iloveimg.com/resize-image/resize-svg.
> ```
> 
> Check our [CONTRIBUTING guide](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards) for more details regarding these errors.
> 
> Please address these issues. When you update this PR, I will check your SVGs again.
> 
> Thanks for your help, SVG-Checker Bot 😄

@yeyeto2788 Please resize the viewbox as explained above. I recommend using [iloveimg](https://www.iloveimg.com/resize-image/resize-svg) to make it fast and easy :)

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Looks good now, but the icons are still quite big. Could you please optimize them using this method? https://github.com/devicons/devicon/issues/1306#issuecomment-1296015861

**Comment by yeyeto2788** [comment]
> Looks good now, but the icons are still quite big. Could you please optimize them using this method? [#1306 (comment)](https://github.com/devicons/devicon/issues/1306#issuecomment-1296015861)

I did that step before my last commit but I have done again hoping it does not screw anything as the last time.

Best.

**Review by Snailedlt — APPROVED** [review]
Looking good! ✔️ 
Thank you for your contribution! 💯

**Review by lunatic-fox — APPROVED** [review]
Approved! ✔
Great contribution! 🚀

**Pull request merged**